### PR TITLE
DO NOT MERGE -- chore(direct): fix directConnect for the last chromedriver downloaded

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by cnishina** [comment]
Should fail since webdriver-manager is not published.

**Comment by cnishina** [comment]
With webdriver-manager@10.2.9: direct connect will read the update-config.json to locate the chromedriver binaries.... will have another PR to fix local driver provider to do similar.

File: circle.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 machine:
+  node:
+    version: 6.9.1
```
Comment (sjelin) [inline circle.yml]: I don't see that version in https://circleci.com/docs/build-image-trusty/#nodejs ?

File: lib/driverProviders/direct.ts [hunk 2]
```diff
@@ -57,15 +55,12 @@ export class Direct extends DriverProvider {
    */
   getNewDriver(): webdriver.WebDriver {
     let driver: webdriver.WebDriver;
+    let updateJson = path.resolve(SeleniumConfig.getSeleniumDir(), 'update-config.json');
```
Comment (sjelin) [inline lib/driverProviders/direct.ts]: We should document this somewhere in wdm

File: circle.yml (hunk 1 above)
Comment (cnishina) [inline circle.yml]: 6.9.1 is the latest stable and Circle CI doesn't complain about it. For some reason I was having issues with spread syntax. This change will also happen when we adopt selenium-webdriver 3.

File: lib/driverProviders/direct.ts (hunk 2 above)
Comment (cnishina) [inline lib/driverProviders/direct.ts]: Yes, I'll create an issue to remind myself in wdm

**Comment by cnishina** [comment]
Since this relies on webdriver-manager (11.0.0) and requires node 6, this PR should be against the beta branch for the next release.

**Pull request closed without merging**